spec/system/sign_index_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe "Sign index", system: true do
```

Comment:
There's an existing spec file that may be a better place for these examples - `my_signs_spec.rb`. It can become confusing if we have multiple files testing the same stuff in two different files.